table/block_based/reader_common.cc

```diff
@@ -33,7 +33,7 @@ Status VerifyChecksum(const ChecksumType type, const char* buf, size_t len,
       actual = XXH32(buf, static_cast<int>(len), 0);
       break;
     case kxxHash64:
-      actual = static_cast<uint32_t>(XXH64(buf, static_cast<int>(len), 0) &
+      actual = static_cast<uint32_t>(XXH64(buf, len, 0) &
```

Comment:
Not a big deal, but we now have a function Lower32of64() that better captures the intent of the static cast on the return value.

Reply:
... which I'm aware you didn't write. Thanks for the fix!